src/AspNet.Security.OAuth.Gitter/project.json

```diff
@@ -0,0 +1,60 @@
+﻿{
+  "version": "1.0.0-beta1-*",
+
+  "description": "ASP.NET Core security middleware enabling Gitter authentication.",
+  "authors": [ "Kévin Chalet", "Jerrie Pelser" ],
+
+  "packOptions": {
+    "owners": [ "Kévin Chalet", "Jerrie Pelser" ],
+
+    "projectUrl": "https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers",
+    "iconUrl": "https://avatars3.githubusercontent.com/u/7998081?s=64",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0.html",
+
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers"
+    },
+
+    "tags": [
+      "aspnetcore",
+      "authentication",
+      "gitter",
+      "oauth",
+      "security"
+    ]
+  },
+
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "nowarn": [ "CS1591" ],
+    "xmlDoc": true
+  },
+
+  "dependencies": {
+    "AspNet.Security.OAuth.Extensions": {
+      "target": "project",
+      "type": "build"
+    },
+    "JetBrains.Annotations": {
+      "type": "build",
+      "version": "10.1.4"
+    },
+    "Microsoft.AspNetCore.Authentication.OAuth": "1.0.0-*"
+  },
+
+  "frameworks": {
+    "net451": { },
+
+    "netstandard1.3": {
+      "dependencies": {
+        "System.Dynamic.Runtime": "4.0.11-*"
```

Comment:
Is this dependency required? (the GitHub provider uses it because it internally relies on `AsJEnumerable()`, but AFAICT, it shouldn't be needed here)